INSTALL

```diff
@@ -1,575 +1,176 @@
-   information about the configure script.
-
- - The "SDL2_image" library.
- 
-   This library loads the PNG images (with the help of libpng) and converts
-   them to "SDL surfaces" that can be handled by the "SDL" library. You may
-   obtain it at:
-
-   http://www.libsdl.org/projects/SDL_image/release/SDL2_image-2.0.1.tar.gz
-
- - The "SDL2_gfx" library.
-
-   http://www.ferzkopp.net/Software/SDL2_gfx/SDL2_gfx-1.0.3.tar.gz
-
- - The "SDL2_ttf" library.
-
-   http://www.libsdl.org/projects/SDL_ttf/release/SDL2_ttf-2.0.14.tar.gz
-
- - The "freetype" library.
-
-   This library helps to render text for the SDL2 client, using an externally
-   provided TrueType font. You may obtain it at:
-
-   http://download.savannah.gnu.org/releases/freetype/freetype-2.7.1.tar.bz2
-
-1d. Prerequisites for the Qt client:
-====================================
+The Freeciv21 project maintains a single Qt client.
 
  - C++ compiler.
 
-   Qt-client is written in C++, so you need appropriate compiler.
-   In Freeciv development, g++ has been used.
-
- - Qt5Core, Qt5Gui, and Qt5Widgets libraries and headers.
-   At least version 5.11 is required.
-
-
-2. Generating Makefiles
-=======================
-This section contains two parts, one for generating makefiles from git
-versions and one for generating makefiles from release versions.
+Qt-client is written in C++, so you need appropriate compiler. In Freeciv21 development, g++ has been used.
 
-2a. Generating the Makefile for git versions:
-=============================================
+ - Qt5Core, Qt5Gui, and Qt5Widgets libraries and headers. At least version 5.11 is required.
 
-This step is only needed for git versions.
 
-To create the makefile just type
+2. Compiling and Installing Freeciv21:
+======================================
 
- % ./autogen.sh
+Compiling Freeciv21 requires the use of the cmake program. cmake does require the use of command line parameters to function
+appropriately.  This section will outline the major use case scenarios.
 
-This will create the configure script and will run it. All parameters
-of autogen.sh are passed to configure. Read the next section about the
-parameters which can be passed to configure.
+First, let's start with describing a few command line parameters. cmake calls the directives and they start with -D.
 
-2b. Generating the Makefile for release versions:
-=================================================
+  -DFREECIV_ENABLE_TOOLS={YES/NO}     -- Enables tools such as the Modpack Installer
+  -DFREECIV_ENABLE_SERVER={YES/NO}    -- Enables the server. Should typically set to YES to be able to play AI games
+  -DFREECIV_ENABLE_NLS={YES/NO}       -- Enabled Native Language Support
+  -DCMAKE_BUILD_TYPE={Release/Debug}  -- Changes the Build Type. Most people will pick Release
+  -DCMAKE_INSTALL_PREFIX=/some/path   -- Allows an alternative install path. Default is /usr/freeciv
+  -DCMAKE_CXX_FLAGS="-O9"             -- Typical CXX Flags to optimize binaries
```

Comment:
This will have no effect if `CMAKE_BUILD_TYPE` is set. The flags are `CMAKE_CXX_FLAGS_RELEASE` etc and have sensible defaults:
```
 CMAKE_CXX_FLAGS                  
 CMAKE_CXX_FLAGS_DEBUG            -g
 CMAKE_CXX_FLAGS_MINSIZEREL       -Os -DNDEBUG
 CMAKE_CXX_FLAGS_RELEASE          -O3 -DNDEBUG
 CMAKE_CXX_FLAGS_RELWITHDEBINFO   -O2 -g -DNDEBUG
 ```

Reply:
Removed from third commit. Not something the average person is going to mess with.

Reply:
Agreed.